tabbar.h

```diff
@@ -17,6 +18,14 @@ namespace Internal {
 class TabBar : public QTabBar
 {
     Q_OBJECT
+
+    struct FullscreenState

```

Comment:
I don't see a reason for this structure to exist, all its elements can be members of `TabBar` class.